template/html/about.html

```diff
@@ -23,4 +23,15 @@ <h3>{{ t "page.about.credits" }}</h3>
     </ul>
 </div>
 
+{{ if .showOptions }}
+<div class="panel">
+    <h3>{{ t "page.about.global_options" }}</h3>
+    <ul>
+    {{ range .options }}
+    <li><strong>{{ .Key }}:</strong> {{ .Value }}</li>
```

Comment:
In some languages, for example in French, there is space before the `:`, but not in English. It might be better to use a table or use a different separator, maybe `=`. The value should probably be surrounded by `<code/>`.